src/data/constants.js

```diff
@@ -0,0 +1,8 @@
+// URL Paths
+export const LOGIN_PAGE = '/login';
+export const REGISTER_PAGE = '/register';
+export const RESET_PAGE = '/reset';
+export const DEFAULT_REDIRECT_URL = '/dashboard';
+
+// Constants
+export const SUPPORTED_ICON_CLASSES = ['apple', 'facebook', 'google', 'microsoft'];
```

Comment:
I believe normally we suggest `fontawesome` classes for `iconClass` while creating a provider configuration. Shouldn't we use the same values here? e.g.  fa-google-plus, fa-facebook, fa-linkedin.

Reply:
In the code Fontawesome icons are used
https://github.com/edx/frontend-app-logistration/blob/cdd36bce1e9f4d8e1bdf39335c527bc18642679b/src/logistration/SocialAuthProviders.jsx#L41-L43

In order to use a string instead of `icon` object I had to pre-add these to the library and use this syntax
```
<FontAwesomeIcon icon=['fab', '<icon_string>'] />
```
The `SUPPORTED_ICON_CLASSES` is the list of icon strings accepted by `fontawesome`